Riot/AppDelegate.m

```diff
@@ -1934,6 +1931,14 @@ - (void)initMatrixSessions
                 }
             }
         }
+        else if ([[UIApplication sharedApplication] applicationState] == UIApplicationStateActive)
+        {
+            if (mxSession.state == MXSessionStateRunning)
+            {
+                // Check if we need to display a key share dialog
+                [self checkPendingRoomKeyRequests];
```

Comment:
You already check [[UIApplication sharedApplication] applicationState] inside the `checkPendingRoomKeyRequests` method

Reply:
true but if you look at the whole state machine management where this piece of code sits, it appears natural to check the UIApplicationStateActive case at this moment.
At least, this is what I feel, no?

Reply:
I understood, but we already know here that the applicationState is not background...